packages/core/js-sdk/src/admin/payment.ts

```diff
@@ -0,0 +1,20 @@
+import { HttpTypes } from "@medusajs/types"
+import { Client } from "../client"
+import { ClientHeaders } from "../types"
+
+export class Payment {
+  private client: Client
+  constructor(client: Client) {
+    this.client = client
+  }
+
+  async capture(id: string, headers?: ClientHeaders) {
```

Comment:
todo: we should add body: `{amount?: number}` to match the API

Reply:
nice, added them!